docs/build.py

```diff
@@ -0,0 +1,73 @@
+"""
+Copyright 2018 JanusGraph Python Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+__author__ = "Debasish Kanhar (https://github.com/debasishdebs)"
+__credits__ = ["Jason Plurad", "sjudeng", "Florian Hockmann", "Misha Brukman"]
```

Comment:
Very kind of you to include me here, but maybe would be best to remove the author/credits/license/email fields entirely?

Reply:
Please check my [comment](https://github.com/JanusGraph/janusgraph-python/pull/4#discussion_r216257759) . 
I certainly can remove it if all of us reach a deciding point :-)